frontend/src/app/pages/browse/browse.component.less

```diff
@@ -39,38 +39,40 @@ div.browse-container {
     
             div.work-header {
                 margin-bottom: 0;
+                   
                 img.rating {
-                    position: relative;
+                    float: left;
```

Comment:
Using the ol' box model here so we can get the short description wrapping around the cover art.